guides/advanced_guides/tokenization.md

```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
 We can break down the tokenization process like so:
 
 1. Crawl the document(s) and determine the primary language for each field.
-2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists. Pipelines include many language-specific operations.
+2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists.
+
+Pipelines include many language-specific operations. Currently, we have two pipelines:
+
+1. A specialized Chinese pipeline using [Jieba](https://github.com/messense/jieba-rs)
+2. A default pipeline that uses [unicode text segmentation](https://github.com/unicode-rs/unicode-segmentation).
+
+For more details or to make a contribution, check out the tokenizer [feature specification](https://github.com/meilisearch/specifications/blob/master/text/0001-script-based-tokenizer.md) and [repository](https://github.com/meilisearch/Tokenizer).
 
 ## Language Support
 
-Currently, the MeiliSearch tokenizer is optimized for four languages:
+**MeiliSearch is multilingual**, featuring optimized support for:
 
-- **English**  🇬🇧
-- **Chinese** 🇨🇳
-- **Japanese** 🇯🇵
-- **Korean** 🇰🇷
+- **Any language that uses whitespace to separate words**
+- **Chinese** 🇨🇳 (through [Jieba](https://github.com/messense/jieba-rs))
 
-It also supports any language that uses the Latin alphabet. **Results may vary in languages with long compound words, such as German.**
+Don't know if your language is covered? **We recommend trying MeiliSearch anyway.** MeiliSearch is a flexible engine designed to deliver results in any language. We are constantly working to improve our multilingual support and [welcome your feedback](/guides/advanced_guides/tokenization.md#future-development).
```

Comment:
I would skip directly to the "feedback welcome part" with something like:

"We aim at providing global language support, and your feedback helps us improving in that sense. If you notice inconsistencies in the way your documents are processed, please open an issue on our [GitHub Repository](link_here)"

Reply:
It really sounds like a trap => "Don't know if your language is covered? **We recommend trying MeiliSearch anyway.**" haha